### PR TITLE
feat(checks): added docs and example for check logo

### PR DESCRIPTION
**Pull request by pon**
@lob/api-engineering-team

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -509,7 +509,8 @@ lob.Check.create(
     to_address = <address_id>,
     bank_account = <bank_account_id>,
     amount = 1000,
-    memo = 'Services Rendered'
+    memo = 'Services Rendered',
+    logo = http://assets.lob.com/lob_check_logo.png
```
Comment (leore) [inline README.md]: wait why is this url in assets.lob.com it shouldnt be in there

Comment (mgartner) [inline README.md]: You need to use `'` here, right?

Comment (pon) [inline README.md]: Damn I didn't notice - I just copied it from the docs page

Comment (leore) [inline README.md]: and that S3 buckets shouldn't have items like this in it

**Comment by pon** [comment]
Updated with new logo link

**Comment by leore** [comment]
lgtm

**Pull request merged**